src/ucp/tag/rndv.c

```diff
@@ -366,14 +366,8 @@ static void ucp_rndv_get_lanes_count(ucp_request_t *rndv_req)
                                                  ep->worker->context->config.ext.max_rndv_lanes);
 }
 
-static ucp_lane_index_t ucp_rndv_get_next_lane(ucp_request_t *rndv_req, uct_rkey_t *uct_rkey)
+static ucp_lane_index_t ucp_rndv_get_lane(ucp_request_t *rndv_req, uct_rkey_t *uct_rkey)
```

Comment:
looks like better - `ucp_rndv_get_zcopy_get_lane()`/`ucp_rndv_retrieve_get_zcopy_lane()` (or smth similar), since the function can't be used to get PUT Zcopy lane.

Reply:
@dmitrygx fixed